### PR TITLE
Added links to workshops

### PR DESCRIPTION
**Pull request by dotemacs**
I've added links to the workshops.

**Don't merge it yet.**

What needs to be done:
- please look at the styling so that it matches the rest of the sections. Some love is needed here.
- add the event ticket in Eventbrite for Biff tutorial
- add the link to this new event in the this PR

This Trello card: https://trello.com/c/yjJDeKAm

### CODE REVIEW

**Comment by reborg** [comment]
I think this looks better?

Screenshot 2020-11-24 at 18 07 05

File: index.html [hunk 1]
```diff
@@ -115,6 +114,46 @@ <h2>Event Schedule</h2>
...
+
+
+          <div role="tabpanel" class="col-lg-9 tab-pane fade show active" id="workshops">
+
+            <div class="row schedule-item">
+              <div class="col-md-3"><time>3:00pm - 6:00pm GMT Thursday 3rd December</time></div>
+              <div class="col-md-9">
+                <h4>re:Crux workshop by JUXT</h4>
+                <p>get your tickets
+                  <a href="https://www.eventbrite.com/e/crux-workshop-reclojure-tickets-129935746315">here</a>
```
Comment (iggar) [inline index.html:134]: I particularly don't like "here" s a link and prefer something more semantic like "at Eventbrite". Or just "grab your ticket" as a link.

File: index.html [hunk 2]
```diff
@@ -115,6 +114,46 @@ <h2>Event Schedule</h2>
...
+              </div>
+            </div>
+
+            <div class="row schedule-item">
+              <div class="col-md-3"><time>9:00am - 10:30am GMT Friday 4th December</time></div>
+              <div class="col-md-9">
+                <h4>Biff workshop by Jacob O'Bryant</h4>
+                <p>get your tickets
+                  <a href="">here</a>
+                </p>
```
Comment (iggar) [inline index.html:146]: missing link (just highlighting this so we don't forget - need to talk to Jacob first?)

**Review by iggar — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (reborg) [reply index.html]: Added link.

File: index.html (hunk 1 above)
Comment (reborg) [reply index.html]: fixed

**Pull request merged**